app/leagues/page.tsx

```diff
@@ -0,0 +1,40 @@
+import {
+  LeagueCard,
+  LeagueCardHeader,
+  LeagueCardTitle,
+  LeagueCardDescription,
+  LeagueCardContent,
+} from '@/components/LeagueCard/LeagueCard';
+
+export default function Leagues() {
+  return (
+    <div>
+      <h1 className="pb-10 text-center text-3xl font-bold leading-[-3%]">
```

Comment:
Why -3%? That's not great when it comes to definite values.

Reply:
@shashilo 3% was the value in the Figma file. I'll use one of tailwinds built-in `leading` classes.

Reply:
Done.